device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt

```diff
@@ -497,18 +497,22 @@ class Simulator (
     override fun approveAccess(bundleId: String) {
         val permissions = SimulatorPermissions(remote, deviceSetPath, this)
 
-        permissions.setServicePermission(bundleId, PermissionType.Camera, PermissionAllowed.Yes)
-        permissions.setServicePermission(bundleId, PermissionType.Microphone, PermissionAllowed.Yes)
-        permissions.setServicePermission(bundleId, PermissionType.Photos, PermissionAllowed.Yes)
-        permissions.setServicePermission(bundleId, PermissionType.Contacts, PermissionAllowed.Yes)
+        val set = PermissionSet()
```

Comment:
PermissionSet is basically a map. but the class name has Set, and variable name is set. maybe some better name ? maybe remove the 'type' from name?